maat/tests/real_world.rs

```diff
@@ -367,29 +385,57 @@ async fn real_world_use_case() {
         )
         .await
         .unwrap();
-    sector
+    let sector = sector
         .prove_commit(
             client.clone(),
             &multi_pair,
-            Arc::new(mapped_parameters),
+            Arc::new(porep_mapped_parameters),
             Arc::new(Semaphore::new(1)),
             CancellationToken::new(),
         )
         .await
         .unwrap();
 
-    // TODO(@th7nder,#615, 21/01/2025): to be fixed in the follow-up for #615.
-    // client.wait_for_height(83, true).await.unwrap();
-    // submit_windowed_post(&client, &charlie_kp).await;
+    // in a network with no sectors, the 1st sector is always assigned to the 1st deadline (index: 0).
+    let deadline = Deadline::new(0, post_proof);
+    let sector_storage = |_sector_number| Some(sector.clone());
```

Comment:
Why are you making a clone, inside a closure, that doesn't even care for the argument it takes?

Reply:
`SectorStorage: Fn(SectorNumber) -> Option<ProvenSector>` -> is the trait bound.
So basically it means that the closure must be callable multiple times. Without the clone, `sector_storage` ends up being `FnOnce`, so a smaller guarantee thatn the `Fn`.